Portfolio Project/index.html

```diff
@@ -9,12 +9,33 @@
 </head>
 <body>
 <header>
+    <a href="#" id="logo">My Logo is beautiful</a>
    <nav>
-        <a href="#" id="logo">My Logo is beautiful</a>
+    <ul class="navbar">
+        <li><a href="#"><i class="fa-solid fa-bars"></i></a></li>
+    </ul>
+     
    </nav>
 </header>
-<h1>Hey there,I'm Alex.</h1>
-<h2>I'm a Software Developer</h2>
-<p>I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don't hestiate to contact me.</p>
+
+<main>  
+    <div id="welcome">
+        <h1>Hey there,I'm Padmnabh</h1>
+        <h2>I'm a Software Developer with 1 year of software development experience</h2>
+        <p>I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don't hestiate to contact me.</p>
+        <ul class="social-media">
+            <li><a href="#"><i class="fa-brands fa-github"></i></a></li>
+            <li><a href="#"><i class="fa-brands fa-linkedin-in"></i></a></li>
+            <li><a href="#"><i class="fa-brands fa-angellist"></i></a></li>
+            <li><a href="#"><i class="fa-brands fa-twitter"></i></a></li>
+            <li><a href="#"><i class="fa-solid fa-m"></i></a></li>
+        </ul>
+    </div>
```

Comment:
- [ ] Please add the correct `margins`, `fonts`, and remove the white border in your design, and try to make it as similar to the Figma template as possible. Cheers 👍  

Figma Design             |  Your Design
:-------------------------:|:-------------------------:
![image](https://user-images.githubusercontent.com/60618877/178121453-4ca7191b-927a-4e77-89e3-62f1efc62096.png)  |  ![image](https://user-images.githubusercontent.com/60618877/178121474-c87fbe6e-80d2-471e-933f-f2d7b06a76e1.png)